Apps/Inc/Tasks.h

```diff
@@ -27,6 +27,7 @@
 #define TASK_READ_SWITCHES_PRIO             7
```

Comment:
Delete the ReadSwitches defns in the tasks.h file

Reply:
this includes all the macros that were defined for it, as well as pruning the data structures that were defined in tasks.c and externed in tasks.h